inputgraph/inputgraph.go

```diff
@@ -271,6 +283,11 @@ func (l *loader) loadTargetFromString(ctx context.Context, targetName string) er
 	return loaderInst.load(ctx)
 }
 
+func (l *loader) hashVersion(v spec.Version) {
```

Comment:
this looks good, can we create a similar function for `hashCommand`

Reply:
Done.